### DISCUSSION TITLE
(Online) rendere for the xml file

### DISCUSSION THREAD

**Discussion by buhtz** [ximion/appstream#581] [Q&A] [answered]
I am new to AppStream and would like to rendere the XML file I do create.
To my understanding its content is used in software centers and app stores. Never used such things and don't want to start with it.
But I want to know how the xml file will get rendered in such a software. This is also important for validation and reviewing the work.

I couldn't find a resource about it.
Might there be an online service where I can look at the rendered result?
If not how can I have a quick look at the rendered result on a Debian system?

**Comment by JakobDev** [comment, marked as answer]
Gnome Software allows you to pass a local file with `--show-metainfo` to show a Preview.

**Reply by buhtz** [reply]
Great. This should be mentioned somehow in the quick start guide.
